db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -1342,6 +1342,12 @@ Status DBImpl::CompactFilesImpl(
                                    newf.second.fd.GetNumber(),
                                    newf.second.fd.GetPathId()));
     }
+
+    for (const auto& blob_file : c->edit()->GetBlobFileAdditions()) {
+      (*output_file_names)
+          .push_back(BlobFileName(c->immutable_options()->cf_paths.front().path,
```

Comment:
Why not output_file_names->push_back?  More common syntax...

Reply:
Yeah I will update it. Thanks!